src/vs/platform/contextkey/browser/contextKeyService.ts

```diff
@@ -46,18 +47,9 @@ export class ContextValuesContainer {
 		}
 		return ret;
 	}
-
-	public fillInContext(bucket: any): void {
-		if (this._parent) {
-			this._parent.fillInContext(bucket);
-		}
-		for (let key in this._value) {
-			bucket[key] = this._value[key];
-		}
-	}
```

Comment:
@joaomoreno I might be missing something but I don't find the place in which now child-context overwrites the values of it's parent. Like, a peek view editor which has `findWidgetVisible: false` and its parent editor/context having `findWidgetVisible: true`